config.yaml

```diff
@@ -22,5 +23,6 @@ tft:
           - name: measure_cpu
           - name: measure_power
           - name: validate_offload
+        secondary_network_nad: "default/ovn-secondary"
```

Comment:
this `config.yaml` is an example, but also maybe something that a user takes as a start for creating their configuration.

The `secondary_network_nand` option and the `oci_bin: docker`) settings don't seem what a user would do by default. Maybe comment these lines out, but leave them here (as comments) to show to the user that those options exist.